azure-tools-common/src/main/java/com/microsoft/azure/common/Utils.java

```diff
@@ -84,4 +87,16 @@ public static String getSegment(String id, String segment) {
         }
         return null;
     }
+
+    public static boolean isPomPackagingProject(String packaging) {
```

Comment:
create an enum for packaging and use fromString here as instead?

Reply:
maven itself uses string for handing packing, it is not necessary to create a enum, since packing is not limited to jar war